Cakes-Co/style.css

```
@@ -1 +1,251 @@
-/*  Add your styling here */
+* {
+    margin: 0;
+    padding: 0;
+  }
+  body {
+    font-family: "Roboto", sans-serif;
+    font-size: 4vw;
+    background-image: linear-gradient(rgb(243, 207, 232), rgb(168, 247, 200), rgb(255, 126, 203));
+     
+  }
+  h1 {
+    font-size: 2.4em;
+    font-weight: 500;
+  }
+  h4 {
+    font-size: 1.2em;
+    font-weight: 400;
+    font-style: italic;
+  }
+
+  h6 {
+    font-size: 0.7em;
+    font-weight: 200;
+  }
+  p {
+    font-size: 1em;
+    font-weight: 300;
+    line-height: 1.4em;
+  }
+  img {
+    width: 100%;
+    border-radius: 10%;
+  }
+  
+  :root {
+    
+    --backgroundColor: linear-gradient(rgb(249, 170, 217), rgb(246, 239, 115), rgb(243, 129, 207));
+    --generalPadding: 1em;
+    --sectionGap: 2em;
+    --white: rgb(75, 6, 52);
+  }
+  
+  header {
+    height: 40vw;
+    padding: var(--generalPadding);
+    background-color: var(--backgroundColor);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: var(--white);
+  }
+  
+  
+
+
+
+  .logo-img {
+    margin-top: var(--generalPadding);
+    height: 70%;
+    width: auto;
+    border-radius: 50%;
+
+  }
+  .header-area {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-end;
+    text-align: end;
+  }
+  #menu-inpt {
+    display: none;
+  }
+  .menu-lbl {
+    height: 30%;
+    width: auto;
+    position: relative;
+    cursor: pointer;
+  }
+
+  .menu-img {
+    height: 100%;
+  }
+
+  .header-area p {
+    width: 19ch;
+  }
+  
+  .nav-nav {
+    display: none;
+    width: min-content;
+    border-radius: var(--generalPadding);
+    padding: var(--generalPadding);
+    background-color: rgb(231, 119, 162);
+    position: absolute;
+    top: 70px;
+    right: 100%;
+    transition: 0s;
+  }
+  
+  #menu-inpt:checked + .nav-nav {
+    display: block;
+    right: var(--generalPadding);
+  }
+  
+  .nav-ul {
```

please consider using html5 semantic element nav and then the unordered list ul such as 
```suggestion
nav ul {
```